universe/ShipPart.cpp

```diff
@@ -47,21 +47,21 @@ namespace {
     }
 
     // create effectsgroup that increases the value of the part meter
-    // of type \a meter_type for part name \a part_name by the fixed
-    // amount \a increase
+    // of type \a meter_type for part name \a part_name
+    // by the result of evalulating \a increase_vr
     std::shared_ptr<Effect::EffectsGroup>
     IncreaseMeter(MeterType meter_type, const std::string& part_name,
-                  float increase, bool allow_stacking = true)
+                  std::unique_ptr<ValueRef::ValueRef<double>>&& increase_vr, bool allow_stacking = true)
     {
         typedef std::vector<std::unique_ptr<Effect::Effect>> Effects;
         auto scope = std::make_unique<Condition::Source>();
         auto activation = std::make_unique<Condition::Source>();
 
         auto value_vr = std::make_unique<ValueRef::Operation<double>>(
-            ValueRef::PLUS,
-            std::make_unique<ValueRef::Variable<double>>(
-                ValueRef::EFFECT_TARGET_VALUE_REFERENCE, std::vector<std::string>()),
-            std::make_unique<ValueRef::Constant<double>>(increase)
+             ValueRef::PLUS,
```

Comment:
why extra indenting?

Reply:
I probably aligned with the variable definition above, will make it four spaces.